packages/charting/src/components/ProgressBarChart/ProgressBarChart.types.ts

```diff
@@ -0,0 +1,129 @@
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
+
+export interface IProgressBarChart {}
+export interface Idata {
```

Comment:
Can you use the IDataPoint under types/IDataPoint, and not repeat making the same interface for every chart? The schema is the same, except it uses x and y in it.